src/orm/base_model.js

```diff
@@ -2385,4 +2404,11 @@ BaseModel.prototype.toJSON = function toJSON() {
   return object;
 };
 
+BaseModel.prototype.isModified = function isModified(propName) {
```

Comment:
Could you please add some documentation with a possible use case for this method too?